versioned_docs/version-8.5/guides/getting-started-java-spring.md

```diff
@@ -49,6 +49,10 @@ If you haven't already, follow [this guide](/self-managed/setup/deploy/local/doc
 1. Use the `docker-compose.yaml` file in [this repository](https://github.com/camunda/camunda-platform).
 2. Clone this repo and run `docker compose up -d` in your terminal to start your environment.
 
+   :::note
+For versions earlier than 8.6, the `docker compose up -d` command is only available to enterprise customers with access to Camunda's private registry. However, from version 8.6 onwards, it will work for all users as Web Modeler images are made publicly available.
+   :::
+
```

Comment:
```suggestion
:::note
For versions earlier than 8.6, the `docker compose up -d` command is only available to enterprise customers with access to Camunda's private registry. However, from version 8.6 onwards, it will work for all users as Web Modeler images are made publicly available.
:::

```